doofinder-for-woocommerce/doofinder-for-woocommerce.php

```diff
@@ -342,12 +344,22 @@ public static function load_only_doofinder_admin_scripts_and_styles() {
             }
         }
 
+        /**
+         * Registers styles used across the admin.
+         *
+         * @return void
+         */
+        public static function register_notices_styles()
+        {
+            wp_enqueue_style('doofinder-notice', Doofinder_For_WordPress::plugin_url() . '/assets/css/doofinder-notice.css');
```

Comment:
I would add the version here since some users have cache plugins like W3 Total Cache or WP Rocket that might cache the old version:

```suggestion
            wp_enqueue_style( 'doofinder-notice', Doofinder_For_WordPress::plugin_url() . '/assets/css/doofinder-notice.css', array(), self::$version );
```

What do you think? :smile: